tools/autodeployment/builds.js

```diff
@@ -124,7 +125,7 @@ function handleBuildByName(buildName) {
 
     // If we have a build process, pipe that, otherwise we're done
     const pipeLog = () => (out ? out.pipe(res) : res.end());
-
+    res.setHeader('Cache-Control', 'no-cache');
```

Comment:
Add this to the Object that has `Content-Type` below (i.e., we already have a spot to include it).